sunpy/coordinates/frames.py

```diff
@@ -81,10 +81,10 @@ class HeliographicStonyhurst(BaseCoordinateFrame):
     ...               frame="heliographic_stonyhurst",
     ...               obstime="2010/01/01T00:00:45")
     >>> sc
-    <SkyCoord (HeliographicStonyhurst: obstime=2010-01-01 00:00:45): (lon, lat, radius) in (deg, deg, km)
+    <SkyCoord (HeliographicStonyhurst: obstime=2010-01-01T00:00:45.000): (lon, lat, radius) in (deg, deg, km)
```

Comment:
I am very interested as to why this changed. I feel like it should not have done. Also I don't really like it.

Reply:
I think it's because `str(Time)` returns depending on the scale and format of `Time`. I think of it more as a feature.

```python
>>> from astropy.time import Time
>>> t = Time.now()
>>> str(t)
'2018-06-05 13:16:28.920780'
>>> str(t.replicate('isot'))
'2018-06-05T13:16:28.921'
>>> str(t.replicate('iso'))
'2018-06-05 13:16:28.921'
>>> str(t.replicate('byear'))
'2018.4275522841342'
>>> str(t.fits)
'2018-06-05T13:16:28.921(UTC)'
```

Reply:
but why was it `.iso` before and now it's `.isot`?

Reply:
It was not `.iso` before, it was `datetime`. `datetime` has the same format as `iso`